test/mocks/upstream/cluster_manager.h

```diff
@@ -105,6 +111,7 @@ class MockClusterManager : public ClusterManager {
   ClusterCircuitBreakersStatNames cluster_circuit_breakers_stat_names_;
   ClusterRequestResponseSizeStatNames cluster_request_response_size_stat_names_;
   ClusterTimeoutBudgetStatNames cluster_timeout_budget_stat_names_;
+  SharedPool::ObjectSharedPool<const envoy::config::cluster::v3::Cluster::CommonLbConfig, MessageUtil, MessageUtil> common_lb_config_pool_;
```

Comment:
This probably needs to be a mock object as it does not have access to a real dispatcher.